docs/api/checkouts.rst

```diff
@@ -1,235 +1,370 @@
+| shipping_address[country]    | request        | Id of the country                                                                                  |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| shipping_address[province]   | request        | *(optional)* Id of the province                                                                    |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[first_name]  | request        | *(optional)* First name for the billing address                                                    |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[last_name]   | request        | *(optional)* Last name for the billing address                                                     |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[city]        | request        | *(optional)* City name                                                                             |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[postcode]    | request        | *(optional)* Postcode                                                                              |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[street]      | request        | *(optional)* Street                                                                                |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[country]     | request        | *(optional)* Id of the country                                                                     |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+| billing_address[province]    | request        | *(optional)* Id of the province                                                                    |
++------------------------------+----------------+----------------------------------------------------------------------------------------------------+
+
+.. note::
+
+    Remember a cart with `id = 21` :doc:from the Cart API documentation </api/carts>? We will take the same cart as an exemplary cart for checkout process.
```

Comment:
🎉  `:doc:from the Cart API documentation </api/carts>` it does not work like that mate ^^